snowpack/src/commands/dev.ts

```diff
@@ -669,7 +668,6 @@ export async function startServer(
       // but we hope to add "virtual file" support soon via plugins. This would
       // be the interface for those response types.
       let foundType = path.extname(reqPath);
-      if (attemptedFileLoc.endsWith('.html')) foundType = '.html';
```

Comment:
Does this work for your usecase instead? This seems to keep the original dev server behavior of falling back to `.html` for requests like `/`:

```ts
      if (!foundType && attemptedFileLoc.endsWith('.html')) foundType = '.html';
```

Reply:
Yes, I’ll test that! And I can see how this reduces the potential impact of the change, without knowing the internals as well as you. 😄

If it works in my tests, would you accept this PR if I could make the changes?

Reply:
My local tests passed, so I’ve gone ahead and made the adjustments in 90e6f30. 😄